nvflare/app_opt/xgboost/histogram_based_v2/adaptor.py

```diff
@@ -345,8 +349,15 @@ def _send_request(self, op: str, req: Shareable) -> bytes:
         Returns:
             operation result
         """
-        reply = self.sender.send_to_server(op, req, self.abort_signal)
+        req.set_header(Constant.MSG_KEY_XGB_OP, op)
+        reply = self.sender.send_to_server(
+            Constant.TOPIC_XGB_REQUEST, req, self.req_timeout, self.fl_ctx, self.abort_signal
```

Comment:
Change to:

with self.engine.new_context() as fl_ctx:
   reply = self.sender.send_to_server(...)

Do not use self.fl_ctx.

Reply:
Changed.